prov/efa/src/rxr/rxr.h

```diff
@@ -710,13 +710,24 @@ struct rxr_ep {
 	struct dlist_entry rx_entry_list;
 	struct dlist_entry tx_entry_list;
 
-	/* number of posted buffer for shm */
-	size_t posted_bufs_shm;
-	size_t rx_bufs_shm_to_post;
-
-	/* number of posted buffers */
-	size_t posted_bufs_efa;
-	size_t rx_bufs_efa_to_post;
+	/*
+	 * number of posted RX packets for shm
+	 */
+	size_t shm_rx_pkts_posted;
```

Comment:
why are we calling them pkts vs bufs now?

Reply:
Because `rx_pkt` is more accurate. We indeed post RX packet to EFA (and shm) to receive data.